suite-native/module-accounts/src/components/TransactionListHeader.tsx

```diff
@@ -32,6 +36,32 @@ type AccountsNavigationProps = TabToStackCompositeNavigationProp<
     AccountsStackParamList
 >;
 
+const GraphlessHeader = ({ accountKey }: { accountKey: AccountKey }) => {
+    const account = useSelector((state: AccountsRootState) =>
+        selectAccountByKey(state, accountKey),
+    );
+
+    if (!account) return null;
+
+    return (
+        <HStack
+            spacing="small"
+            flexDirection="row"
+            alignItems="center"
+            marginBottom="large"
+            justifyContent="center"
+        >
+            <CryptoIcon name={account.symbol} />
+            <CryptoAmountFormatter
+                value={account.balance}
+                network={account.symbol}
+                isBalance={false}
+                adjustsFontSizeToFit
+            />
+        </HStack>
+    );
```

Comment:
You could create component TransactionListHeaderBalance or something like this and use it for testnet and normal coins. This feels like it's duplication of parts of AccountDetailGraphHeader component. You could then use the component there so when we need to restyle it, you only need to change one file so it's less error prone

Reply:
Applied in f5d488e1f.